wwwbase/admin/bulkReplace.php

```diff
@@ -61,60 +54,43 @@
   Session::set('objCount', $objCount);
   Session::set('objChanged', 0);
   Session::set('objExcluded', 0);
-  Session::set('objStructured', []);
+  Session::set('objRemainingIds', $objRemainingIds);
+  Session::set('objStructuredIds', []);
+  Session::set('phase', 0);
   Session::set('finishedReplace', false);
 }
 
 // variables should not be null
+$phase = Session::get('phase');
 $objCount = Session::get('objCount');
 $objChanged = Session::get('objChanged');
 $objExcluded = Session::get('objExcluded');
-$objStructured = Session::get('objStructured');
-
-// preparing the main query object with global parameters
-$query = $query
-       ->order_by_asc('id')
-       ->limit($limit);
+$objRemainingIds = Session::get('objRemainingIds');
```

Comment:
Aș vota pentru niște redenumiri de variabile (în PHP și în sesiune), că mi-a luat un pic să înțeleg ce face fiecare. De exemplu, aș pune `count` sau `cnt` sau `num` în cele care țin valori numerice: `numMatched`, `numRemaining`, `numExcluded`, `numChanged` (și documentat undeva că, prin contract, `numMatched = numRemaining + numExcluded + numChanged` la orice moment). Și similar pentru vectorii de ID-uri.

Putem face asta într-un PR separat, că altfel înverzim tot fișierul. :-) Dar n-aș lăsa-o nefăcută.

Reply:
Sunt pentru. Eu folosesc notări `intCountExcluded`, `arrExcludedIds`, `boolSomething`, de exemplu pentru a ști ce tip de variabilă este. Nu țin neapărat la int, arr etc., mă pliez cerințelor, doar să facem o convenție (sau există notat undeva?).

Reply:
Prefixarea cu tipul pare un pic prea prolixă. În general te poți lămuri din context: dacă e `countSomething` sau `numSomething` e întreg, dacă e `somethingIds` (la plural) e vector, dacă `isSomething` sau `hasSomething` e boolean... Dar nu țin morțiș la opinia asta. Cum crezi și tu!

Reply:
Ah, nu! Ziceai că într-un PR separat.